### ISSUE TITLE
Tabs a11y issue: status of each tab selected is not being read out

### ISSUE THREAD

**Issue by tinayuangao** [angular/material2#7934]
#### Bug, feature request, or proposal:
Bug

#### What is the expected behavior?
under tabs link, status of each tab selected should be read out.

#### What is the current behavior?
under tabs link, status of each tab selected is not being read out. user has to navigate forward / backward and come back to selected tab to be notified of the action performed. 

#### What are the steps to reproduce?
1. Click on link 'tabs'                                                                                                                                                                                                                                                       2. Navigate to heading ' dog breeds '                                                                                                                                                                                                                                                       3. Navigate to ' labrador retriever ' tab and click enter keystroke.                                                                                                                                                                                                                                                  3. Observe physically the tab is being selected but screen reader does not announce the selection or No earcon is observed .                                                                    Note: Observe the same behavior for tabs displayed under weather heading.


#### What is the use-case or motivation for changing an existing behavior?


#### Which versions of Angular, Material, OS, TypeScript, browsers are affected?
Chromevox next (Chrome OS 62.0.3202.43 Official build dev 64 bit)  , Voiceover (mac OS version: 10.12.6) , JAWS (18.0.4350) 

#### Is there anything else we should know?

**Comment by M-a-c** [comment]
@tinayuangao command-option + space, seems to be working
Voiceover (mac OS version: 10.12.6) chrome and safari

**Comment by josephperrott** [comment]
Closing as this a11y issue has been resolved since this issue was reported.  Working as expeceted as of 7.0.0 release.
